mh/pat_item.py

```diff
@@ -783,6 +1042,18 @@ def getHunterStats(hr=921, profile=b"Navaldeus",
      - 0xf7: City's seeking
      - 0xf8: Server type
     """
+    """
+    Analysis of HunterSettings packet:
```

Comment:
This function is a dummy function to provide a dummy HunterStats. You should avoid messing with it and create a proper function and avoid passing as much as parameters because it's error prone. You can put this docstring analysis under this new function, especially if it doesn't provide new insights compared to what's already described just above.

Reply:
Reverted in an upcoming commit.